tests/manage/z_cluster/nodes/test_node_kernel_crash_ceph_fsync.py

```diff
@@ -0,0 +1,114 @@
+import logging
+import random
+import pytest
+
+from time import sleep
+from ocs_ci.ocs import constants, node
+from ocs_ci.framework.testlib import E2ETest, tier2, bugzilla, polarion_id
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException
+from ocs_ci.ocs.node import get_worker_nodes
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2075068")
+@polarion_id("OCS-3947")
+class TestKernelCrash(E2ETest):
+    """
+    Tests to verify kernel crash
+    """
+
+    pvc_size = 1
+    result_dir = constants.MOUNT_POINT + "/mydir"
+
+    def creates_files(self, pod_obj):
+        while True:
+            pod_obj.exec_sh_cmd_on_pod(
+                "for i in {1..125}; do dd if=/dev/zero of=%s/emp_$i.txt bs=1M count=1; done"
```

Comment:
Please set a variable for this as you might want to change it to something other than 125 in the future